### PR TITLE
[VL] Add Celeborn support in doc

### PR DESCRIPTION
**Pull request by kerwin-zk**
## What changes were proposed in this pull request?

Add Celeborn support in doc

### CODE REVIEW

**Comment by kerwin-zk** [comment]
@zhouyuan @zhztheplayer cc

File: docs/Velox.md [hunk 1]
```diff
@@ -298,6 +298,33 @@ If you are using instance credentials you do not have to set the access key or s
...
+## 2.6 Celeborn support
+
+Gluten with velox backend supports [Celeborn](https://github.com/apache/incubator-celeborn) as remote shuffle service. Below introduction is used to enable this feature
+
+First refer to this URL(https://github.com/apache/incubator-celeborn) to setup a celeborn cluster.
+
+Currently to use Celeborn following configurations are required in spark-defaults.conf
+```
+spark.shuffle.manager org.apache.spark.shuffle.celeborn.CelebornShuffleManager
+spark.serializer org.apache.spark.serializer.KryoSerializer
```
Comment (zhouyuan) [inline docs/Velox.md:310]: is this necessary for gluten?

Comment (kerwin-zk) [reply docs/Velox.md:310]: No need, already removed.

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**